packages/patternfly-4/react-core/src/components/Nav/NavItem.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import styles from '@patternfly/patternfly/components/Nav/nav.css';
+import { css } from '@patternfly/react-styles';
+import { Omit } from '../../helpers/typeUtils';
+import { NavContext } from './Nav';
+
+export interface NavItemProps extends Omit<React.HTMLProps<HTMLAnchorElement>,  'onClick'> {
+  /** Content rendered inside the nav item */
+  children?: React.ReactNode | string | null;
```

Comment:
I think `React.ReactNode` covers the `| string | null`. Does removing it causes a problem?